### ISSUE TITLE
TheTVDB data needs sanitizing for leading spaces

### ISSUE THREAD

**Issue by mhurwood** [TV-Rename/tvrename#652]
There are quite a few "bad" episode titles in TheTVDB's database which have leading spaces embedded in them. This results in TVRename naming a file something like:

The Show 1x01 -    Episode Name.mp4

i.e. extra spaces before "Episode" and after the dash, in this naming scheme.

There aren't any circumstances under which you'd want to keep the leading spaces (that I can think of), and cleaning up the "bad" data in TheTVDB is laborious (if you even notice the problem!) so it'd be good to have them removed automatically. (Maybe make a log entry, so you can still check for episodes to clean up in the source data if desired.)

**Comment by SirSparkles** [comment]
Any specific shows I can test it on?

**Comment by mhurwood** [comment]
Sure, an example is The Simpsons, Episode 30x20 "I'm Just a Girl Who Can't Say D'oh" which gets a single space before the "I'm".

Found a similar problem with trailing spaces **after** the show name too, very hard to fix on TheTVDB I believe as you basically have to submit a request to change a show name.  An example is "Blue Murder; Killer Cop" Show (ID 331913)

I just did some searching of the TheTVDB.XML file and found lots of examples of "sloppy spaces".
I'd be tempted to have a LTRIM(RTIM()) done to most string fields coming down from TheTVDB database, but I have particularly found examples in the show name, episode title, overview, actor's name and actor's role fields.
